features/alt-text-in-content.yml

```
@@ -0,0 +1,6 @@
+name: Alt text in content
+description: "Add alternate text in the CSS `content` property after a `/`. For example, `content: url('upvote.svg') / 'Upvote'` will have the accesible text 'Upvote'."
```

Trying to work this into our usual "X does Y" construction, rather than an imperative and future tense.

Also some tweaks to make more explicit what's going on with the example.

```suggestion
description: "The `/` notation in `content` CSS property values adds alternative text to generated content. For example, the `content: url('upvote-icon.svg') / 'Upvote'` declaration adds accessible "Upvote" text to the SVG icon. Also known as alternate text."
```